docs/reference/run.md

```diff
@@ -201,6 +202,14 @@ more advanced use case would be changing the host's hostname from a container.
 > **Note**: `--uts="host"` gives the container full access to change the
 > hostname of the host and is therefore considered insecure.
 
+## INIT settings (--init)
+   Default to docker standard method of running containers.
```

Comment:
Defaults to Docker's standard method of running contianers.

Though this is still confusing.